coderd/database/queries/organizations.sql

```diff
@@ -39,6 +39,7 @@ WHERE
 
 -- name: InsertOrganization :one
 INSERT INTO
-	organizations (id, "name", description, created_at, updated_at)
+	organizations (id, "name", description, created_at, updated_at, is_default)
 VALUES
-	($1, $2, $3, $4, $5) RETURNING *;
+	-- If no organizations exist, and this is the first, make it the default.
+	($1, $2, $3, $4, $5, (SELECT count(*) FROM organizations) = 0) RETURNING *;
```

Comment:
Performance isn't going to be an issue here, but `(SELECT TRUE FROM organizations LIMIT 1) IS NULL` would also work and only needs to check one row.

Reply:
I'll update :+1: